docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,160 @@
+# Bitbucket integration
+
+!!! note
+    <!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse's Bitbucket integration is new and still in Beta:
+
+    -   For now, we only show the [Accelerate metrics](../metrics/accelerate.md) using your merged pull requests to automatically detect deployments.
+    -   <!--NOTE Limitation from Bitbucket integration V1-->At the moment, we're only collecting new data, so you won’t see any metrics after completing the integration setup. We'll automatically import historical data and backfill your charts within a few weeks.
+
+<!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse integrates with Bitbucket Cloud to receive data about changes and deployments, necessary to calculate the metrics:
+
+-   [Lead time for changes](../metrics/accelerate.md#lead-time-for-changes)
+
+-   [Deployment frequency](../metrics/accelerate.md#deployment-frequency)
+
+## Setting up the Bitbucket integration
+
+!!! note
+    -   The Bitbucket integration connects to a specific [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). You must be the **owner** of that Bitbucket workspace to proceed with the integration setup.
+    -   [See below](#bb-permissions) the permissions that Pulse requires from your Bitbucket account.  
+
+To set up the Bitbucket integration:
+
+1.  On Pulse, [expand **Integrations** and select **Bitbucket**](https://app.pulse.codacy.com/integrations/bitbucket){: target="_blank"}.
+
+1.  Click **Install Bitbucket App**.
+
+    ![Installing the Pulse Bitbucket App](images/bb-installing.png)
+
+1.  Choose the workspace that you want to connect the integration to and confirm with **Complete setup**. You can only select a workspace where you have owner permissions.
```

Comment:
```suggestion
1.  Choose the workspace that you want to connect the integration to and click **Complete setup** to confirm. You can only select a workspace where you have owner permissions.
```

Reply:
OK, thanks! I updated as suggested and cropped the screenshots to match the dimensions of the GitHub ones.